packages/okta-react/README.md

```diff
@@ -1,9 +1,29 @@
+[Okta Auth SDK]: https://github.com/okta/okta-auth-js
+[react-router]: https://github.com/ReactTraining/react-router
+[higher-order component]: https://reactjs.org/docs/higher-order-components.html
+[Auth service]: #auth
+
 # Okta React SDK
 
 [![npm version](https://img.shields.io/npm/v/@okta/okta-react.svg?style=flat-square)](https://www.npmjs.com/package/@okta/okta-react)
 [![build status](https://img.shields.io/travis/okta/okta-oidc-js/master.svg?style=flat-square)](https://travis-ci.org/okta/okta-oidc-js)
 
-Okta React SDK makes it easy to integrate [react-router](https://github.com/ReactTraining/react-router) with Okta's [OpenID Connect API](https://developer.okta.com/docs/api/resources/oidc.html).
+Okta React SDK builds on top of the [Okta Auth SDK][]. This SDK adds integration with [react-router][] and provides additional logic and components designed to help you quickly add authentication and authorization to your React single-page web application.
+
+With the [Okta Auth SDK][], you can:
+
+- Login and logout from Okta using the [OAuth 2.0 API](https://developer.okta.com/docs/api/resources/oidc)
+- Retrieve user information
+- Determine authentication status
+- Validate the current user's session
+
+All of these features are supported by this SDK. Additionally, using this SDK, you can:
+
+- Add "secure" routes, which will require authentication before render
+- Enter a custom login flow when authentication is required
```

Comment:
I feel "Enter a custom login flow" can be confusing to some users. (Considering that we have IdX pipeline changes) We can probably say - Provide customizable self-hosted login page for authentication or something like that.

Reply:
This is referring to our `onAuthenticationRequired` callback function. They can do almost anything with this function, but our samples typically redirects to a specific `login` route.

Reply:
This does read really generic, might be better to be direct, e.g. "Define custom logic/behavior when authentication is required"